CMakeLists.txt

```diff
@@ -434,17 +434,17 @@ function(add_tarball targetname namever treeish)
 	set(distname ${tarname}.bz2)
 	set(docname ${namever}-doc.${distfmt})
 
-	add_custom_target(${docname}
-		DEPENDS man apidoc
+	file(GLOB man_pages docs/man/*.[1-8])
```

Comment:
What's the reason for this? AFAICS the tarball gets updated on man page changes as it is, through the "man" target.